lib/utils.js

```diff
@@ -16,15 +17,37 @@ module.exports.extractKVBindings = (setFlagValues) => {
     const value = kvComponents.slice(1).join('=')
 
     if (!bindings[variable]) {
-      bindings[variable] = new KeyValueStore()
+      bindings[variable] = new KeyValueStore(filepaths[variable])
     }
 
     bindings[variable].put(key, value)
   }
 
+  for (const [variable, path] of Object.entries(filepaths)) {
+    bindings[variable] = new KeyValueStore(path)
```

Comment:
I believe this should only be done if a binding for the variable doesn't exist yet. I could be wrong but as it's currently written, all variables that have key-values set and have a file path will be overwritten with an empty KeyValueStore.

Reply:
Yes, good point - this was supposed to come before the other loop!